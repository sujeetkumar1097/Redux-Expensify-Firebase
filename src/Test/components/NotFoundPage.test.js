import React from 'react'
import {shallow} from 'enzyme'
import NotFoundPage from '../../components/PageNotFound'

test('Not Found Page Render Correctly', () =>{
    const wrapper = shallow(<NotFoundPage/>)
    expect(wrapper).toMatchSnapshot()
})