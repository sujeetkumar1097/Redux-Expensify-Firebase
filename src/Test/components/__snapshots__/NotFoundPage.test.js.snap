// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not Found Page Render Correctly 1`] = `
<h1>
  Inside 404! PageNotFound
</h1>
`;
